pkgs/development/python-modules/google-auth/default.nix

```diff
@@ -29,8 +29,12 @@ buildPythonPackage rec {
     cachetools
     pyasn1-modules
     rsa
-    pyopenssl
     pyu2f
+  ] ++ lib.optionals (!(stdenv.isDarwin && stdenv.isAarch64)) [
```

Comment:
Please stop doing dependencies platform specific if they really aren't. Either we drop this for all platforms or aarch64-darwin is out of luck. I can't test this when updating on linux and when the dependency changes in the future it will break silenetly.